utils/utils.py

```diff
@@ -369,3 +372,13 @@ def write_to_csv_by_chunks(file_to_write, output_file, write_mode="wb", chunksiz
                 buffer = f.read(chunksize)
         else:
             raise NotImplementedError(f"file' extension: [{file_to_write}] is not supported yet")
+
+
+def check_support(current_params, unsupported_params):
+    ignored_params = {}
+    for param in unsupported_params:
+        if current_params.get(param):
```

Comment:
```suggestion
        if current_params.get(param) is not None:
```
otherwise you won't be warning on `0` or `False` which might have a meaning